web/webpack.config.js

```diff
@@ -14,6 +14,7 @@ const config = {
     path: path.join(__dirname, 'dist'),
     filename: 'app.js'
   },
+  // devtool: 'cheap-module-eval-source-map',
```

Comment:
Useful in dev for enabling breakpoints and debugging in Chrome (and VSCode, and probably Atom too!), but breaks the build.  Probably ought to configure it to only enable in dev builds, but it's not instantly clear to me how to do that so it can wait.

Reply:
👍